package.json

```diff
@@ -38,7 +40,9 @@
     "image"
   ],
   "dependencies": {
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "react": "^15.5.4",
```

Comment:
As long as it's in one or the other, no sense bikeshedding about it.  Sorry, if I'd seen the entries in devDependencies I never would have made this change, don't want to get into a bikeshed conversation ;)